Source/Task/ThreadPool_stl.cpp

```diff
@@ -43,6 +47,11 @@ class ThreadPoolImpl
                 numThreads--;
                 m_pool.emplace_back(std::thread([this]
                 {
+#if defined(HC_PLATFORM) && HC_PLATFORM == HC_PLATFORM_ANDROID
+                    JNIEnv* jniEnv = nullptr;
+                    JavaVM* jvm = s_javaVm;
```

Comment:
>s_javaVm [](start = 34, length = 8)

it feels like you should be initializing this to null here

Reply:
yes, remainder from a previous iteration where I only read `s_javaVm` once